Modules/_asynciomodule.c

```diff
@@ -406,15 +406,11 @@ future_ensure_alive(FutureObj *fut)
 static int
 future_schedule_callbacks(asyncio_state *state, FutureObj *fut)
 {
-    Py_ssize_t len;
-    Py_ssize_t i;
-
     if (fut->fut_callback0 != NULL) {
         /* There's a 1st callback */
 
-        int ret = call_soon(state,
-            fut->fut_loop, fut->fut_callback0,
-            (PyObject *)fut, fut->fut_context0);
+        int ret = call_soon(state, fut->fut_loop, fut->fut_callback0,
+                            (PyObject *)fut, fut->fut_context0);
```

Comment:
I can't see what's changing here - is it just formatting?

Reply:
I added a comment about an evil call soon before but observed that it wasn't relevant so I removed it. Since I was anyway changing the function, I made it PEP-7 but I can revert that part though.

Reply:
Please revert. 
The project's policy is: don't touch the code just for reformatting, only make required changes.
It helps keeping the history better discoverable while the code doesn't look as good as it could, I agree.